crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -1319,6 +1358,58 @@ impl AuthorityPerEpochStore {
             .pending_checkpoint_signatures
             .insert(&(checkpoint_seq, index), info)
     }
+
+    pub(crate) fn record_epoch_pending_certs_process_time_metric(&self) {
+        if let Some(epoch_close_time) = *self.epoch_close_time.read() {
+            self.metrics
+                .epoch_pending_certs_processed_time_since_epoch_close_ms
+                .report(epoch_close_time.elapsed().as_millis() as u64);
+        }
```

Comment:
Should this panic or print errors when `epoch_close_time` is `None`?

Reply:
epoch_close_time can be None if the validator crashed after closing epoch.